contracts/subdaos/cwd-subdao-timelock-single/src/contract.rs

```diff
@@ -309,18 +318,41 @@ fn is_overrule_proposal_rejected(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn reply(deps: DepsMut, _env: Env, msg: Reply) -> Result<Response, ContractError> {
+pub fn reply(deps: DepsMut, env: Env, msg: Reply) -> Result<Response, ContractError> {
     let proposal_id = msg.id;
 
     PROPOSALS.update(deps.storage, proposal_id, |prop| match prop {
         Some(mut prop) => {
             prop.status = ProposalStatus::ExecutionFailed;
-
             Ok(prop)
         }
         None => Err(ContractError::NoSuchProposal { id: proposal_id }),
     })?;
 
+    // Error is reduced before cosmwasm reply and is expected in form of "codespace=? code=?"
+    PROPOSAL_FAILED_EXECUTION_ERRORS.update::<_, ContractError>(
+        deps.storage,
+        proposal_id,
+        |maybe| {
+            let error = msg
+                .result
+                .into_result()
+                .err()
+                .unwrap_or_else(|| "result is not error".to_string());
```

Comment:
if `result` is not an error that means something goes really wrong and we got a good result instead? How that can even happen? Maybe we should not process such cases and just panic?